osu.Framework/Input/InputManager.cs

```diff
@@ -107,7 +107,7 @@ public Drawable DraggedDrawable
         /// Top-down in this case means reverse draw order, i.e. the front-most visible
         /// <see cref="Drawable"/> first, and <see cref="Container"/>s after their children.
         /// </summary>
-        public IReadOnlyList<Drawable> HoveredDrawables => hoveredDrawables;
+        public IReadOnlyList<Drawable> HoveredDrawables => hoveredDrawables.AsSlimReadOnly();
```

Comment:
```suggestion
        public SlimReadOnlyListWrapper<Drawable> HoveredDrawables => hoveredDrawables.AsSlimReadOnly();
```
That should fix the remaining allocation.

Reply:
Hmm, right. Didn't realise we were happy with exposing these lists with the wrapper, but that *is* how it's done everywhere else.